saleae/saleae.py

```diff
@@ -211,16 +211,16 @@ def __init__(self, host='localhost', port=10429, quiet=False, args=None):
 			self._s.connect((host, port))
 		except ConnectionRefusedError:
 			log.info("Could not connect to Logic software, attempting to launch it now")
-			Saleae.launch_logic(quiet=quiet, args=args)
+			Saleae.launch_logic(quiet=quiet, host=host, port=port, args=args)
 
 		try:
 			self._s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 			self._s.connect((host, port))
 		except ConnectionRefusedError:
-			print("Failed to connect to saleae at {}:{}".format(host, port))
-			print("")
-			print("Did you remember to 'Enable scripting socket server' (see README)?")
-			print("")
+			log.info("Failed to connect to saleae at {}:{}".format(host, port))
```

Comment:
This should probably actually be all the way to a `log.warn` or really `log.error` I think

Reply:
ACK.